extensions/refresh-catalog/src/main/java/org/eclipse/dataspaceconnector/mvd/FederatedCacheNodeResolver.java

```diff
@@ -0,0 +1,66 @@
+/*
+ *  Copyright (c) 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.mvd;
+
+import org.eclipse.dataspaceconnector.catalog.spi.FederatedCacheNode;
+import org.eclipse.dataspaceconnector.iam.did.spi.document.DidDocument;
+import org.eclipse.dataspaceconnector.iam.did.spi.document.Service;
+import org.eclipse.dataspaceconnector.iam.did.spi.resolution.DidResolverRegistry;
+import org.eclipse.dataspaceconnector.registration.client.models.Participant;
+import org.eclipse.dataspaceconnector.spi.monitor.Monitor;
+import org.eclipse.dataspaceconnector.spi.result.Result;
+
+import java.util.List;
+import java.util.Optional;
+
+import static java.lang.String.format;
+
+/**
+ * Resolves the {@link FederatedCacheNode}s from the Participant's Did Document.
+ */
+class FederatedCacheNodeResolver {
+
+    public static final String IDS_MESSAGING = "IDSMessaging";
+    public static final List<String> IDS_MULTIPART = List.of("ids-multipart");
```

Comment:
there is a constant in EDC: `Protocols.IDS_MULTIPART`, it's in the `ids-spi` module. Please use that constant here. Then, the list variable should be named `protocols`

Reply:
done